src/ixdat/plotters/tpms_plotter.py

```diff
@@ -146,139 +141,452 @@ def plot_measurement(
+                ax.legend
+
+            return axes if axes else ax
+
+        else:
+            self.ms_plotter.plot_vs(
                 measurement=measurement,
                 ax=ax,
-                axes=axes,
                 mass_list=mass_list,
-                mass_lists=mass_lists,
                 mol_list=mol_list,
-                mol_lists=mol_lists,
                 tspan=tspan,
                 tspan_bg=tspan_bg,
                 remove_background=remove_background,
                 unit=unit,
                 x_unit=x_unit,
-                x_name=T_name,
+                x_name=inverse_T_name,
                 logplot=logplot,
                 logdata=logdata,
                 legend=legend,
                 **kwargs,
             )
+            if fit_arrhenius:
+                t, T = measurement.grab(
+                    inverse_T_name,
+                    tspan=tspan,
+                    include_endpoints=True
+                )
+                if mol_list:
+                    t_v, V = measurement.grab_flux(
+                    mol_list[0],
+                    tspan=tspan,
+                    include_endpoints=False,
+                    )
+                elif mass_list:
+                    t_v, V = measurement.grab(
+                    mass_list[0],
+                    tspan=tspan,
+                    include_endpoints=False,
+                    )
 
-        return axes
+                T_mol = np.interp(t_v, t, T)
+                #v_avg_list = np.log(np.array(v_avg_list))
+                coef = measurement.fit_to_arrhenius_equation(
+                    inverse_T=T_mol,
+                    k=np.log(V),
+                    logdata=True,
+                )
+                k_fitted = coef[1] + coef[0] * T_mol
+                logplot=logplot
+                if not logdata:
+                    logplot=True
+                    # values for lotting arrheniuos
+                    popt, pcov = measurement.fit_to_arrhenius_equation(
+                        inverse_T=T_mol,
+                        k=V,
+                        logdata=logdata
+                    )
+                    k_fitted = measurement._func(T_mol, *popt)
+                    # log arrhen np.log(v_avg_list) = - Ea / R * 1/ T_avg_list + np.log(A)
 
-    def plot_in_one_fig(
+                ax.plot(
+                        T_mol,
+                        k_fitted,
+                        color=fit_arrhenius_color,
+                        label='arrhenius',
+                        linestyle='--',
+                        )
+            if logplot:
+                ax.set_yscale("log")
+            if legend:
+                ax.legend
+
+        return ax
+
+    def plot_measurement_in_one_panel(
         self,
         *,
         measurement=None,
         ax=None,
         axes=None,
         mass_list=None,
-        mass_lists=None,
         mol_list=None,
-        mol_lists=None,
+        meta_list=None,
```

Comment:
See my comment in `TPMSPlotter.plot_measurement` about meta_list(s). I think there is a simpler way!